### PR TITLE
uplink-notunnel: simplify creation of static macaddress

### PR DESCRIPTION
**Pull request by SvenRoederer**
use the functions provided by OpenWrt to format the address and set to "LAA"-bit.

* was tested in an separate script 
* looks more elegant than the loop before

### CODE REVIEW

File: uplinks/freifunk-berlin-uplink-no-tunnel/uci-defaults/freifunk-berlin-notunnel [hunk 1]
```diff
@@ -27,11 +29,8 @@ uci set network.ffuplink_dev.type=veth
 uci set network.ffuplink_dev.name=ffuplink
 uci set network.ffuplink_dev.peer_name=ffuplink_wan
 # Create a static macaddr starting with "fe" for ffuplink devices
```
Comment (pmelange) [inline uplinks/freifunk-berlin-uplink-no-tunnel/uci-defaults/freifunk-berlin-notunnel]: You might want to remove/change this comment.

**Comment by pmelange** [comment]
The same mac-loop is in migration.  Maybe it would make sense to add the change to migration in this PR.

**Comment by pmelange** [comment]
I just saw that there is another problem in the migration script.  A small but important typo.

line 413
```
uci set network.ffuplink_dev_macaddr=$macaddr
```
should be 
```
uci set network.ffuplink_dev.macaddr=$macaddr
```

**Comment by SvenRoederer** [comment]
> I just saw that there is another problem in the migration script. A small but important typo.

so just push a fix ...

**Pull request merged**